i18n/testdata/exports/two_questions.po

```diff
@@ -7,6 +7,7 @@ msgstr ""
 "Language: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
+"Language-3: \n"
```

Comment:
Should we include empty headers?  Don't feel strongly either way but feels slightly weird.

Reply:
I noticed that POEdit always includes all official headers with blank values.. so 🤷‍♂